src/test/controller_preset_validation_test.cpp

```diff
@@ -74,6 +74,7 @@ class FakeController : public Controller {
     }
 
   protected:
+    using Controller::send;
```

Comment:
What is the purpose of using here?

Reply:
It brings the function of the superclass into scope. Otherwise FakeController::send hides it.

Reply:
Can't we just rename one send()

Reply:
Ok, nevermind it is just a test case.

Reply:
Ohhh now I see the problem. With this using we are importing a pure virtual overload of send. Thanks! I was looking for the problem on the wrong class, the error message is quite misleadindg :(

Reply:
Here I have removed the using, but it didn't fix the problem.
https://github.com/ferranpujolcamins/mixxx/tree/script_updates

Reply:
> the error message is quite misleadindg :(

Welcome to C++.